### DISCUSSION TITLE
Please help sass warning for angular 17.0.9

### DISCUSSION THREAD

**Discussion by ghost** [twbs/bootstrap#39579] [Q&A]
[WARNING] Deprecation [plugin angular-sass]

    src/styles/bootstrap/_variables.scss:662:37:
      662 │ ...-group-margin:                 ($grid-gutter-width / 2) !default;

**Comment by patkon** [comment]
I think you need to use `calc`

```...-group-margin: calc($grid-gutter-width / 2) !default;```
